### PR TITLE
parameters written as fully expanded YAML dictionaries

### PR DESCRIPTION
**Pull request by arbabnazar**
@edx/devops kindly review. Thanks

### CODE REVIEW

File: playbooks/create_django_ida.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 # Creates a new ansible role
 # Usage:
-#   ansible-playbook ./create_django_ida.yml -i "localhost," -c local -e role_name=my_awesome_role
+#   ansible-playbook ./create_django_ida.yml -i "localhost," -c local -e my_role_name=my_awesome_role
```
Comment (arbabnazar) [inline playbooks/create_django_ida.yml:4]: reason to change the `role_name` into `my_role_name` that `role_name` was introduced as magic/predefined variable in 2.x  while it was not in 1.9. Thanks

**Comment by e0d** [comment]
:+1: 

@clintonb not this affects the IDA role

@edx/devops second look.

**Comment by clintonb** [comment]
Noted.

**Comment by feanil** [comment]
:+1:

**Pull request merged**